.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: python
-python: 2.7
+python: 
+    - 2.7
+    - 3.5
 addons:
   firefox: 47.0.1
 env:
-- TOXENV=tests DISPLAY=:99.0
+- TOXENV=tests DISPLAY=:99.0 FILES="tests/*.py tests/webdriver/*.py"
+- TOXENV=marionette DISPLAY=:99.0 FILES=tests/marionette/*.py
 - TOXENV=flake8
+matrix:
+  allow_failures:
+    - env: TOXENV=marionette DISPLAY=:99.0 FILES=tests/marionette/*.py
```

Comment:
We only want to allow failures against python 3.5, which we can do by naming the environments differently and using [tox-travis](https://pypi.python.org/pypi/tox-travis). I'll explain in the following comments.